packages/shared/src/hooks/useFeedLayout.ts

```diff
@@ -58,6 +58,10 @@ export const FeedLayoutMobileFeedPages = new Set([
   OtherFeedPage.SearchBookmarks,
   OtherFeedPage.UserUpvoted,
   OtherFeedPage.UserPosts,
+  OtherFeedPage.Explore,
+  OtherFeedPage.ExploreLatest,
+  OtherFeedPage.ExploreDiscussed,
+  OtherFeedPage.ExploreUpvoted,
```

Comment:
Asking again here. Why did we end up moving these feed pages to `OtherFeedPage`. The reason we had it at `SharedFeedPage` is to understand that they are shared across both webapp and extension.

https://github.com/dailydotdev/apps/pull/3237#discussion_r1645933777

Reply:
BTW I didn't know SharedFeedPage had that purpose as well to differentiate for what is shared in the extension.

So SharedFeedPages are what is checked for wether to redirect to /onboarding, those pages are what is behind auth. 

If we want to keep them at SharedFeedPages then we just need to filter the /posts out when checking for authenticaiton

Reply:
No they are SEO pages so highly important they are shown to bots and not redirect 🙏

Reply:
Oh. So we are back to allowing feeds to anonymous users now?

Reply:
@sshanzel Not all, just the explore ones as they are for SEO purposes mainly (addition people can visit them yeah)
(Same for tag page where we removed the auth banner as well for instance)

Reply:
Sorry for not getting back to u on this Lee, see u already asked product :D